### ISSUE TITLE
use "case count()" , the graphd will be crashed

### ISSUE THREAD

**Issue by Codelone** [vesoft-inc/nebula-graph#1160]
nebula-graph 2.0.1 version

`YIELD case COUNT($-.var_idcards) ==100  when true THEN 1 ELSE 0 END as aa`
Expect an error, but the graphd crashed !

**Comment by HarrisChu** [comment]
```
YIELD 99 as var_idcards | YIELD case COUNT($-.var_idcards) ==100 when true THEN 1 ELSE 0 END as aa
```

graphd-stderr.log

```
PC: @          0x1a5edcd _ZNSt17_Function_handlerIFvPN6nebula7AggDataERKNS0_5ValueEEZNS0_18AggFunctionManagerC4EvEUlS2_S5_E0_E9_M_invokeERKSt9_Any_dataOS2_S5_
*** SIGSEGV (@0xa0) received by PID 10425 (TID 0x7fdc2edfe700) from PID 160; stack trace: ***
    @          0x1e66491 (unknown)
    @     0x7fdc3494262f (unknown)
    @          0x1a5edcd _ZNSt17_Function_handlerIFvPN6nebula7AggDataERKNS0_5ValueEEZNS0_18AggFunctionManagerC4EvEUlS2_S5_E0_E9_M_invokeERKSt9_Any_dataOS2_S5_
    @          0x136a0f6 nebula::AggregateExpression::apply()
    @          0x136a8b9 nebula::AggregateExpression::eval()
    @          0x1352570 nebula::RelationalExpression::eval()
    @          0x1372a97 nebula::CaseExpression::eval()
    @          0x123ab19 nebula::graph::ProjectExecutor::execute()
    @          0x1327bce nebula::graph::Scheduler::execute()
    @          0x132a96b _ZN5folly6detail8function14FunctionTraitsIFvONS_3TryIN6nebula6StatusEEEEE9callSmallIZNS_7futures6detail10FutureBaseIS5_E18thenImplementationINS4_5graph9Scheduler8ExecTaskIZNSH_10doScheduleEPNSG_8ExecutorEEUlS5_E3_EENSC_14callableResultIS5_SM_EELb0EJOS5_EEENSt9enable_ifIXsrNT0_13ReturnsFutureE5valueENSR_6ReturnEE4typeEOT_NSC_9argResultIXT1_ESW_JDpT2_EEEEUlS7_E_EEvRNS1_4DataES7_
    @          0x10d3969 _ZN5folly6detail8function14FunctionTraitsIFvvEE9callSmallIZNS_7futures6detail4CoreIN6nebula6StatusEE10doCallbackEvEUlvE_EEvRNS1_4DataE
    @          0x10d3f0b folly::futures::detail::Core<>::doCallback()
    @          0x132784b _ZN5folly6detail8function14FunctionTraitsIFvONS_3TryISt6vectorIN6nebula6StatusESaIS6_EEEEEE9callSmallIZNS_7futures6detail10FutureBaseIS8_E18thenImplementationIZNS5_5graph9Scheduler18doScheduleParallelERKSt3setIPNSJ_8ExecutorESt4lessISN_ESaISN_EEEUlS8_E_NSF_14callableResultIS8_SU_EELb0EJOS8_EEENSt9enable_ifIXntsrNT0_13ReturnsFutureE5valueENSZ_6ReturnEE4typeEOT_NSF_9argResultIXT1_ES14_JDpT2_EEEEUlSA_E_EEvRNS1_4DataESA_
    @          0x132c8e1 _ZN5folly6detail8function14FunctionTraitsIFvvEE9callSmallIZNS_7futures6detail4CoreISt6vectorIN6nebula6StatusESaISB_EEE10doCallbackEvEUlvE_EEvRNS1_4DataE
    @          0x132d9be folly::futures::detail::Core<>::doCallback()
    @          0x132efe3 std::_Sp_counted_ptr_inplace<>::_M_dispose()
    @          0x109c3b7 std::_Sp_counted_base<>::_M_release()
    @          0x132c0b7 _ZN5folly6detail8function9execSmallIZNS_7collectIN9__gnu_cxx17__normal_iteratorIPNS_6FutureIN6nebula6StatusEEESt6vectorIS9_SaIS9_EEEEEENS6_ISB_INSt15iterator_traitsIT_E10value_type10value_typeESaISJ_EEEESG_SG_EUlONS_3TryIS8_EEE_EEbNS1_2OpEPNS1_4DataEST_
    @          0x10ceb2a folly::futures::detail::Core<>::CoreAndCallbackReference::detach()
    @          0x10d4021 folly::futures::detail::Core<>::doCallback()
    @          0x10d409c folly::futures::detail::Core<>::doCallback()
    @          0x132a8d3 _ZN5folly6detail8function14FunctionTraitsIFvONS_3TryIN6nebula6StatusEEEEE9callSmallIZNS_7futures6detail10FutureBaseIS5_E18thenImplementationINS4_5graph9Scheduler8ExecTaskIZNSH_10doScheduleEPNSG_8ExecutorEEUlS5_E3_EENSC_14callableResultIS5_SM_EELb0EJOS5_EEENSt9enable_ifIXsrNT0_13ReturnsFutureE5valueENSR_6ReturnEE4typeEOT_NSC_9argResultIXT1_ESW_JDpT2_EEEEUlS7_E_EEvRNS1_4DataES7_
    @          0x10d3969 _ZN5folly6detail8function14FunctionTraitsIFvvEE9callSmallIZNS_7futures6detail4CoreIN6nebula6StatusEE10doCallbackEvEUlvE_EEvRNS1_4DataE
    @          0x10d3f0b folly::futures::detail::Core<>::doCallback()
    @          0x132784b _ZN5folly6detail8function14FunctionTraitsIFvONS_3TryISt6vectorIN6nebula6StatusESaIS6_EEEEEE9callSmallIZNS_7futures6detail10FutureBaseIS8_E18thenImplementationIZNS5_5graph9Scheduler18doScheduleParallelERKSt3setIPNSJ_8ExecutorESt4lessISN_ESaISN_EEEUlS8_E_NSF_14callableResultIS8_SU_EELb0EJOS8_EEENSt9enable_ifIXntsrNT0_13ReturnsFutureE5valueENSZ_6ReturnEE4typeEOT_NSF_9argResultIXT1_ES14_JDpT2_EEEEUlSA_E_EEvRNS1_4DataESA_
    @          0x132c8e1 _ZN5folly6detail8function14FunctionTraitsIFvvEE9callSmallIZNS_7futures6detail4CoreISt6vectorIN6nebula6StatusESaISB_EEE10doCallbackEvEUlvE_EEvRNS1_4DataE
    @          0x132d9be folly::futures::detail::Core<>::doCallback()
    @          0x132efe3 std::_Sp_counted_ptr_inplace<>::_M_dispose()
    @          0x109c3b7 std::_Sp_counted_base<>::_M_release()
    @          0x132c0b7 _ZN5folly6detail8function9execSmallIZNS_7collectIN9__gnu_cxx17__normal_iteratorIPNS_6FutureIN6nebula6StatusEEESt6vectorIS9_SaIS9_EEEEEENS6_ISB_INSt15iterator_traitsIT_E10value_type10value_typeESaISJ_EEEESG_SG_EUlONS_3TryIS8_EEE_EEbNS1_2OpEPNS1_4DataEST_
    @          0x10ceb2a folly::futures::detail::Core<>::CoreAndCallbackReference::detach()
    @          0x10d4021 folly::futures::detail::Core<>::doCallback()
```

**Comment by CPWstatic** [comment]
Fixed by #1012
